src/server/pps/server/pipeline_controller.go

```diff
@@ -434,10 +434,10 @@ func (pc *pipelineController) apply(ctx context.Context, pi *pps.PipelineInfo, r
 	return nil
 }
 
-// RcIsFresh returns a boolean indicating whether rc has the right labels
+// rcIsFresh returns a boolean indicating whether rc has the right labels
```

Comment:
I think this function was being exported before your changes. Do we need to lock it down?

Reply:
I think it was unexported before based on the diff here:
https://github.com/pachyderm/pachyderm/blob/707a081ac581286d758ba4732aaedd409a926a8c/src/server/pps/server/pipeline_controller.go